### PR TITLE
Ci enable client linter

### PR DESCRIPTION
**Pull request by touilleMan**
# What has changed ?

- [ ] Does this PR affect the User ?

### CODE REVIEW

File: oxidation/client/.eslintrc.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ module.exports = {
     'cypress'
   ],
   rules: {
-    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-console': 'off',
+    'no-debugger': 'error',
```
Comment (FirelightFlagboy) [inline oxidation/client/.eslintrc.js:21]: Would be noisy during debug session, Why not keeping the system where a env variable can disable this rule ?

File: oxidation/client/package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
         "test:unit": "vue-cli-service test:unit",
         "test:e2e": "vue-cli-service test:e2e",
         "test:e2e:headless": "vue-cli-service test:e2e --headless",
-        "lint": "vue-cli-service lint",
+        "lint": "vue-cli-service lint  --max-errors 0 --max-warnings 0",
```
Comment (FirelightFlagboy) [inline oxidation/client/package.json:11]:
```suggestion
        "lint": "vue-cli-service lint --max-errors 0 --max-warnings 0",
```
:stuck_out_tongue:

Comment (touilleMan) [reply oxidation/client/package.json:11]: 👍

**Comment by NicoTuxx** [comment]
no longer relevant since we moved to Vite

**Pull request closed without merging**